galite-core/src/main/kotlin/org/kopi/galite/form/VField.kt

```diff
@@ -1590,41 +1592,35 @@ abstract class VField protected constructor(width: Int, height: Int) : VConstant
    * !!! TRY TO MERGE WITH checkList ???
    */
   fun getListID(): Int {
-    TODO()
-    /*val SELECT_IS_IN_LIST = " SELECT  ID                      " +
-            " FROM    $2                      " +
-            " WHERE   $1 = $3"
 
     assert(!isNull(block!!.activeRecord)) { threadInfo() + " is null" }
     assert(list != null) { threadInfo() + "list ist not null" }
```

Comment:
list is not null

Reply:
list should not be null be is nullable , shouldve been fixed with kotlin non nullable type instead